Cargo.toml

```diff
@@ -6,11 +6,19 @@ edition = "2018"
 license = "Apache-2.0"
 description = "Utility functions for generating Rust code from protobufs (using protobuf-rust or Prost)"
 
+[features]
+default = ["protobuf-codec"]
+protobuf-codec = ["protobuf-codegen", "protobuf"]
+grpcio-protobuf-codec = ["grpcio-compiler/protobuf-codec", "protobuf-codec"]
```

Comment:
Why separate out the grpcio thing into a feature? Is it ever used without?

Reply:
For example, raft-proto don't need grpcio.